### ISSUE TITLE
Fyi on xcode/macos about 32bit sunset

### ISSUE THREAD

**Issue by thezoggy** [sabnzbd/sabnzbd#1093]
https://arstechnica.com/gadgets/2018/01/apple-prepares-macos-users-for-discontinuation-of-32-bit-app-support/

**Comment by thezoggy** [comment]
I think we still include 32bit utilities.. we prob could stop that officially soon and drop legacy osx support..

**Comment by sanderjo** [comment]
@thezoggy Do you run macOS High Sierra 10.13.4 beta yourself? If so, can you confirm you get the message? Do you get a message if you run a CLI-only command? 

As they say: "Meten is weten", so in the .dmg let's check for 32-bit executables:

```
sander@sammie-1710:/media/sander/SABnzbd-2.3.2$ find . -executable -type f -print0 |  xargs -0 file | grep  "386 executable"
./SABnzbd.app/Contents/Resources/osx/unrar/unrar:                                                                                             Mach-O i386 executable, flags:<NOUNDEFS|DYLDLINK|TWOLEVEL|WEAK_DEFINES|BINDS_TO_WEAK>
```
So only the unrar executable is 32bit. However, https://www.rarlab.com/download.htm and https://www.rarlab.com/rar_add.htm do not provide a 64-bit MacOS version ...

**Comment by thezoggy** [comment]
@sanderjo I do not run macos, but i know that we relied on 32bit utilities for mac.. I just didnt know which off the top of my head. Glad to hear that we could prob just compile and include our own version.. hopefully it 'works' without any special requirements.

**Comment by Safihre** [comment]
Bit weird that Unrar supplies only a 32bit official download.. But indeed we can always compile it ourselves in case.

**Comment by Safihre** [comment]
Closed by #1097
